### PR TITLE
[15.0][ADD] event_registration_mail_compose

### PR DESCRIPTION
**Pull request by ivantodorovich**
Allow to send emails to multiple attendees at once, using the mail composer.

Same feature [proposed to odoo/master](https://github.com/odoo/odoo/pull/83801)

### CODE REVIEW

File: event_registration_mail_compose/README.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
```
Comment (petrus-v) [inline event_registration_mail_compose/README.rst:2]: not sure moka CI ig generating the readme, probably we have to generate ourself or decide to not generate or keep it empty ??

**Review by petrus-v — APPROVED** [review]
LGTM !

File: event_registration_mail_compose/README.rst (hunk 1 above)
Comment (petrus-v) [reply event_registration_mail_compose/README.rst:2]: sorry I figure out we are on an OCA repo....

**Review by chienandalu — APPROVED** [review]
Thanks @ivantodorovich !

**Pull request merged**